src/Symfony/Component/Scheduler/Schedule.php

```diff
@@ -62,6 +70,17 @@ public function remove(RecurringMessage $message): static
         return $this;
     }
 
+    /**
+     * @return $this
+     */
+    public function removeById(string $id): static
```

Comment:
Not used?

Reply:
Not here but I thought it might be useful to be able to remove it directly by id (can be used easily in a listener that listens to the 'pre_run' event e.g)